tests/system/action/poll/test_create.py

```diff
@@ -102,3 +105,299 @@ def test_create_three_options(self) -> None:
         assert option_4.get("used_as_global_option_in_poll_id") == 1
         assert option_4.get("meeting_id") == 112
         assert option_4.get("weight") == 1
+
+
+class CreatePollOS3TestCase(BaseActionTestCase):
```

Comment:
I miss tests for the auto-creation of the global option.

Reply:
test_create_correct checks the autocreating of a global option